src/components/keyboard/Write.tsx

```diff
@@ -23,19 +23,21 @@ const keyboardRef = useRef(null)
         );
     }
 
-    function onChangeInput(event:any) {
+    function onChangeInput(event: any) {
```

Comment:
```suggestion
    function onChangeInput(event: React.ChangeEvent<HtmlInputElement>) {
```
or maybe ChangeEvent without React, since it is used in a html element rather than a react component
```